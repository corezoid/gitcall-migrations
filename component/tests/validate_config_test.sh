#!/usr/bin/env bash
$CZ component generate-config ../ > "${CZ_TMP_DIR}/generated_config.yaml"
$CZ ops valid-config "${CZ_TMP_DIR}/generated_config.yaml" ../schema/main.json ../schema
