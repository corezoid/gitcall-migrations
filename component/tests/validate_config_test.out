level=error msg="gitcall_migrations.config.db: String length must be greater than or equal to 1"
level=error msg="gitcall_migrations.config.host: String length must be greater than or equal to 1"
level=error msg="gitcall_migrations.config.migrator_password: String length must be greater than or equal to 1"
